components/Meta.js

```diff
@@ -4,8 +4,6 @@ import Reset from './style/Reset'
 import Font from './style/Font'
 import Typography from './style/Typography'
 
-import '../static/react-crop.css'
```

Comment:
I want to move all the custom themes to `/static` and just use `<links>`, removing `@zeit/next-css`